### DISCUSSION TITLE
Character Roles

### DISCUSSION THREAD

**Discussion by JElgar** [White-Rabbit-Entertainment/game#83] [Ideas]
@isaacwood 

-Captain - > navigation of ship 
-First Mate/ Quartermaster - > navigation of ship if captain unavailable, in charge of food and supplies 
-Sailing Master -> in charge of navigation and plotting. E.g looking at a map and relaying to captain 
-Gunner -> operate the cannons 
-Powder Monkey - > help gunners 
-Cooks -> prepare and cook the food in the kitchen 
-Boatswain -> prepare ship for battle / do repairs. Dropping anchor, swab deck 
-Cooper -> make and maintain barrels. 
-Musicians -> play songs (potential for Alice?).

**Comment by isaacwood** [comment]
With Character Roles, we could try and localize where the Agents can go on the ship by only allowing them to interact with a subset of interactables. E.g a gunner agent will only interact with the cannons and cannon balls.
